legal-api/src/legal_api/reports/business_document.py

```diff
@@ -545,6 +547,29 @@ def _set_amalgamating_details(self, business: dict):
             # (if a business was a TED and it is now a TING). No need to show amalgamating businesses information
             business['amalgamatedEntities'] = []
 
+    def _set_amalgamation_out(self, business: dict):
+        filings = Filing.get_filings_by_types(self._business.id, ['amalgamationOut'])
+        if filings:
+            filing = filings[0]
+            filing_info = {}
+            filing_meta = filing.meta_data
+            filing_type = filing.filing_type
+            filing_info['filingName'] = BusinessDocument._get_summary_display_name(filing_type, None, None, None)
+
+            country_code = filing_meta['amalgamationOut']['country']
+            region_code = filing_meta['amalgamationOut']['region']
+
+            country = pycountry.countries.get(alpha_2=country_code)
+            region = None
+            if region_code and region_code.upper() != 'FEDERAL':
+                region = pycountry.subdivisions.get(code=f'{country_code}-{region_code}')
+            filing_info['jurisdiction'] = f'{region.name}, {country.name}' if region else country.name
+            filing_info['foreignLegalName'] = filing_meta['amalgamationOut']['legalName']
+            amalgamation_out_date = LegislationDatetime.as_legislation_timezone_from_date_str(
+                filing_meta['amalgamationOut']['amalgamationOutDate'])
+            filing_info['amalgamationOutDate'] = amalgamation_out_date.strftime(OUTPUT_DATE_FORMAT)
+            business['amalgamationOut'] = filing_info
+
```

Comment:
Move this to https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/business_document.py#L322